test/e2e/network_segmentation.go

```diff
@@ -32,9 +32,12 @@ import (
 )
 
 const openDefaultPortsAnnotation = "k8s.ovn.org/open-default-ports"
+const RequiredUDNNamespaceLabel = "k8s.ovn.org/user-defined-primary-network"
 
 var _ = Describe("Network Segmentation", func() {
 	f := wrappedTestFramework("network-segmentation")
+	// disable automatic namespace creation, we need to add the required UDN label
+	f.SkipNamespaceCreation = true
```

Comment:
Commenting here but it related to other places as well, can we do the same as in egressip, label the namespace we got from the infra?

Reply:
Not a must, can be done in follow-up PR

Reply:
cant, because the label must exist when the namespace is created. The validating admission policy wont let you add it later.